### PR TITLE
Update deps to kubernetes v0.7.2.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
cc @rjnagal

**Comment by rjnagal** [comment]
go vet has moved. You nedd to update travis. We already did that for cadvisor yesterday.

**Comment by vishh** [comment]
I have sent another PR to update go vet.

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -27,48 +27,48 @@
 		},
 		{
 			"ImportPath": "github.com/GoogleCloudPlatform/kubernetes/pkg/api",
-			"Comment": "v0.7.0-256-g7dec65f",
-			"Rev": "7dec65f535ad95cdc1f88a672d6880f0720c8241"
+			"Comment": "v0.7.0-13-gff7eb21",
```
Comment (rjnagal) [inline Godeps/Godeps.json]: How do we tell if v0.7.0-13-* is moving forward from v0.7.0-256? Shouldn't it be v0.7.2-smth?

Comment (vishh) [inline Godeps/Godeps.json]: I don't know. I am intending to fix the deps to v0.7.2.

**Comment by vishh** [comment]
Ping @rjnagal. Travis is happy now :)

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**